whitepaper/chapters/system.tex

```diff
@@ -4,20 +4,20 @@ \section{System}
 \nemquote{
 }{}
 
-\nemchapterfirstletter{H}{igh} degrees of customization are supported by \codenamespace at both the network level and the individual node level.
-Network wide settings specified in \nemsetting{network}{network} must be the same for all nodes in a network.
-Node specific settings specified in \nemsetting{node}{node} can vary across nodes in the same network.
+\nemchapterfirstletter{C}{atapult} supports high customization at both the network and individual node levels.
+Network-wide settings must be the same for all nodes in a network, and are specified in \nemsetting{network}{network}.
```

Comment:
consider: 
Network-wide settings, specified in \nemsetting{network}{network}, must be the same for all nodes in a network.

Reply:
Finally kept "subset of configuration", sounds more natural.